src/Sentry.Bindings.Cocoa/buildTransitive/Sentry.Bindings.Cocoa.targets

```diff
@@ -49,4 +49,15 @@
     </ItemGroup>
   </Target>
 
+  <!--
+   Ensure the AppDomain.CurrentDomain.UnhandledException gets triggered - see:
+    https://github.com/xamarin/xamarin-macios/issues/15252
+
+   Not supported on the CoreCLR or NativeAOT - see:
+    https://github.com/xamarin/xamarin-macios/blob/0a81da28e4217624a54ccb83541e1eb8ba651fca/runtime/runtime.m#L2480-L2491
+  -->
+  <PropertyGroup Condition="'$(UseMonoRuntime)' == 'true'">
```

Comment:
We should probably also restrict this to net8.0 since this issue has been [fixed in net9.0 on MONO](https://github.com/xamarin/xamarin-macios/issues/15252#issuecomment-2349301905)